molgenis-navigator/src/main/frontend/src/store/actions.js

```diff
@@ -89,6 +89,24 @@ function queryPackages (query: string) {
   })
 }
 
+/**
+ *
+ * Validating specific input for searchbox
+ *
+ * @param query query to send to api/v2
+ */
+function validateQuery (query: string) {
```

Comment:
Can't this function be more generic? I like the clear error message but you never know what people might put in this query and if it is not in this function, they will still receive an error message like: {}